cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,58 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+/// <reference types="cypress" />
+/// <reference types="../support" />
+
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const signInPage = new SignInPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.register();
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
```

Comment:
The email and password used here are hardcoded and do not match the generated user data. Consider using `user.email` and `user.password` to ensure consistency with the generated user.